### PR TITLE
Feat/integrations

### PR DESCRIPTION
**Pull request by Darkheir**
* Changed Title "Intelligence Center External Integrations" to "External Integrations" so it applies better to the TIP context
* Added the Anomali integration in the CTI part only

### CODE REVIEW

**Comment by Darkheir** [comment]
Anomali page can be found here: https://storage.gra.cloud.ovh.net/v1/AUTH_741841e1c7da4c50a76152cbc2609bd1/sekoiaio-documentation-previews/397/cti/features/integrations/anomali/

File: _shared_content/intelligence_center/integrations/anomali.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# External Integrations: Anomali ThreatStream
+
+SEKOIA's CTI feed is available in [Anomali's market place](https://www.anomali.com/marketplace/threat-intelligence-feeds), in the Threat Intelligence Feeds category
+
+Our integrations allows to see all the objects our feed produces along with the relationships between them.
+The PDF's of our reports can be downloaded in the attachment tab. 
+
+All our indicators are created as observables in the platform, and the sources are positionned as tags along with the kill chain phases.
+
+![Anomali's feed](/assets/intelligence_center/anomali_feed.png){: style="width: 100%; max-width: 100%"}
```
Comment (otetard) [inline _shared_content/intelligence_center/integrations/anomali.md:10]:
```suggestion
![!Anomali's feed](/assets/intelligence_center/anomali_feed.png){: style="width: 100%; max-width: 100%"}
```

Small trick to allow users to open images in fullscreen.

**Review by otetard — DISMISSED** [review]

File: _shared_content/intelligence_center/integrations/anomali.md (hunk 1 above)
Comment (Darkheir) [reply _shared_content/intelligence_center/integrations/anomali.md]: Nice one :-)

**Comment by Darkheir** [comment]
In the preview I can't see the image when in full screen, is it normal ?

**Comment by otetard** [comment]
> In the preview I can't see the image when in full screen, is it normal ?

Yes, this is due to the way we handle paths in previews (unfortunately).

**Review by otetard — APPROVED** [review]

**Pull request merged**